patterns/nvidia-gpu-efa/generate-efa-info-test.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+export MPI_JOB_NAME=efa-info-test
```

Comment:
these are all known values so lets just make it a static yaml file that users can apply